lib/compiler/graph-builder.js

```diff
@@ -304,6 +307,10 @@ var GraphBuilder = Base.extend({
         }
 
         return null;
+    },
+    _semantic_ast: function(ast) {
+        var semantic = new SemanticPass({ now: this.now });
```

Comment:
This is a bit of defensive move as there are no guarantees about the state of the `SemanticPass` instance after one runs `sa_expr` which really wasn’t intended as a public API. I didn’t want to attack that problem now as I hope to solve more generally when I’ll make `SemanticPass` a visitor.